build.sbt

```diff
@@ -4,7 +4,7 @@ import sbt._
 val PlayVersion = "2.5.4"
 val Scala10 = "2.10.5"
 val Scala11 = "2.11.8"
-val ProjectVersion = "0.2.2"
+val ProjectVersion = "0.2.3"
```

Comment:
@slavaschmidt, tests were not passing until I updated version. It seems that `sbt scripted` uses published version instead of local-published.